File: miranaSolution.Data/Entities/Book.cs
namespace miranaSolution.Data.Entities
{
    public class Book
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string ShortDescription { get; set; }
        public string LongDescription { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        public string ThumbnailImage { get; set; }
        public string Slug { get; set; }

        public int AuthorId { get; set; }
        public Author Author { get; set; }

        public Guid UserId { get; set; }
        public AppUser AppUser { get; set; }

        public List<Chapter> Chapters { get; set; }

        public List<BookGenre> BookGenres { get; set; }

        public List<Rating> Ratings { get; set; }
    }
}

File: src/frontend/miranaSolution.WebApp/src/helpers/models/catalog/books/Book.ts
export interface Book {
    id: number;
    name: string;
    shortDescription: string;
    longDescription: string;
    createdAt: Date;
    updatedAt: Date;
    thumbnailImage: string;
    isRecommended: boolean;
    slug: string;
    authorName: string;
    authorId: number;
    genres: string[];
    isDone: boolean;
    viewCount: number;
}